packages/front-end/components/Experiment/TabbedPage/HealthTab.tsx

```diff
@@ -86,12 +85,24 @@ export default function HealthTab({
     [onDrawerNotify]
   );
 
-  // If org has not updated settings since the health tab was introduced, prompt the user
-  // to enable the traffic query setting
+  const noExposureQueryMessage = `The health tab only works when your experiment has an Exposure
```

Comment:
nit: If this string is always the same for `noExposureQueryMessage` pull it out outside of the component (above line 28) You'll also want to use `"` instead of `` ` ``